crates/ott-balancer-bin/src/config.rs

```diff
@@ -56,6 +56,31 @@ impl BalancerConfig {
         // SAFETY: get is never called before CONFIG is initialized.
         unsafe { CONFIG.as_ref().expect("config not initialized") }
     }
+
+    pub fn set() -> &'static Self { 
+        figment::providers::Jail::expect_with(|jail| {
+            jail.create_file("Fly.toml", r#"
+                name = "Just a TOML App!"
+                count = 100
+            "#)?;
+        
+            jail.create_file("Fly.json", r#"
+                {
+                    "name": "Just a JSON App",
+                    "authors": ["figment", "developers"]
+                }
+            "#)?;
+        
+            jail.set_env("FLY_REGION", 250);
+        
+            // Sources are read _eagerly_: sources are read as soon as they are
+            // merged/joined into a figment.
+            let figment = figment::Figment::new()
+                .merge(figment::providers::Toml::file("Fly.toml"))
+                .merge(figment::providers::Env::prefixed("FLY_"))
```

Comment:
We can't accept all `FLY_` environment variables because it opens us up to configuration collision -- Fly could add a new environment var that overlaps with a config option, and that would be bad,